src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -0,0 +1,119 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h" /* Defines HAVE_RDMACM_QP_LESS */
+#endif
+
+#if HAVE_RDMACM_QP_LESS
+
+#include "rdmacm_cm.h"
+#include <ucs/async/async.h>
+
+#include <poll.h>
+#include <rdma/rdma_cma.h>
+
+UCS_CLASS_INIT_FUNC(uct_rdmacm_listener_t, uct_cm_h cm,
+                    const struct sockaddr *saddr, socklen_t socklen,
+                    const uct_listener_params_t *params)
+{
+    return UCS_ERR_NOT_IMPLEMENTED;
+}
+
+UCS_CLASS_CLEANUP_FUNC(uct_rdmacm_listener_t){}
+
+UCS_CLASS_DEFINE(uct_rdmacm_listener_t, uct_listener_t);
+UCS_CLASS_DEFINE_NEW_FUNC(uct_rdmacm_listener_t, uct_listener_t,
+                          uct_cm_h , const struct sockaddr *, socklen_t ,
+                          const uct_listener_params_t *);
+UCS_CLASS_DEFINE_DELETE_FUNC(uct_rdmacm_listener_t, uct_listener_t);
+
+static void uct_rdmacm_cm_cleanup(uct_cm_h cm)
```

Comment:
uct_rdmacm_cm object could also be implemented as a CLASS

and maybe separate rdmacm_cm and rdmacm_listener to different source files?